packages/video_player/video_player/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: video_player
 description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
-version: 0.10.9+1
+version: 0.10.10
```

Comment:
Does this PR change the public api of the plugin?
We usually only update minors when there are non-breaking public api changes.

Reply:
It doesn't change the API.  It isn't a breaking change in that sense.  It adds no new functionality from the users perspective.

Reply:
if it doesn't add anything from users perspective, we should change it to 0.10.9+2 so users 
who depends on `video_player: ^0.10.9^` can get this update.

Reply:
My understanding is that the "+N" nomenclature is for changes that don't change the binary.  This doesn't change the intention of the code, but the binary is definitely changed.